tests/unit/h/services/annotation_stats_test.py

```diff
@@ -1,160 +1,114 @@
-from unittest import mock
+from unittest.mock import call, sentinel
 
 import pytest
 
-from h.search import (
-    DeletedFilter,
-    Limiter,
-    Search,
-    TopLevelAnnotationsFilter,
-    UserFilter,
-)
 from h.services.annotation_stats import AnnotationStatsService, annotation_stats_factory
 
 
 class TestAnnotationStatsService:
-    def test_total_user_annotation_count_calls_search_with_request(
-        self, svc, search, pyramid_request
+    def test_user_annotation_count(
+        self, pyramid_request, svc, Search, TopLevelAnnotationsFilter
     ):
-        svc.total_user_annotation_count("userid")
+        num_annotations = svc.user_annotation_count(sentinel.userid)
```

Comment:
Using `num_annotations` as the variable names in these tests, instead of `anns`.

Also things like `sentinel.userid` and `sentinel.pubid` instead of `"userid"` and `"groupid"`